lib/scrabble.rb

```diff
@@ -1,7 +1,33 @@
+require "pry"
 class Scrabble
 
+  attr_reader :word
+
+  def initialize
+    @word = word
+  end
```

Comment:
`@word` will always be `nil` because `word` is not passed in as an argument, so will always evaluate to `nil`.